src/main/java/co/rsk/federate/signing/hsm/message/PowHSMSignerMessageBuilder.java

```diff
@@ -59,7 +62,8 @@ public SignerMessage buildMessageForIndex(int inputIndex) throws SignerMessageBu
                 inputIndex,
                 txReceipt,
                 receiptMerkleProof,
-                sigHash
+                sigHash,
+                outpointValues
```

Comment:
If we are always passing a list of outpoint values, then is the constructor of PowHSMMessage that does not receive the outppint values used anywhere?

Reply:
It is not used anymore in the `main/java` but in some tests in  `test/java`. Thinking of keeping the PR small I just thought getting rid of the use of that constructor could be done in another task. Do you think is ok to do it in this PR?

Reply:
Done.